.vscode/launch.json

```diff
@@ -4,6 +4,7 @@
     {
       "name": "Import tests",
       "program": "${workspaceFolder}/server/scripts/import-tests/index.js",
+      // "args": ["-c","5fe7afd82fe51c185b8661276105190a59d47322"],
```

Comment:
Thanks for highlighting this, it does look like commented code I forgot to remove 😜 But actually I left it in intentionally because I think it will be useful.

The launch.json enables us to use the VSCode launcher to debug the script that imports the tests from the ARIA-AT repo. However it wasn't very clear how to import a specific commit, which is something we do when creating test environments. This commented code will be useful to anyone who needs to debug the import of a specific commit.